master/locals.tf

```diff
@@ -0,0 +1,24 @@
+# ------------------------------------------------------------------------------
+# Retrieve the information for all accouts in the organization.  This is used
+# to lookup the account IDs for all assessment accounts.
+# ------------------------------------------------------------------------------
+data "aws_organizations_organization" "cool" {
+}
+
+# ------------------------------------------------------------------------------
+# Evaluate expressions for use throughout this configuration.
+# ------------------------------------------------------------------------------
+locals {
+  # Look up all assessment account IDs from AWS organizations provider
+  assessment_account_ids = [
+    for account in data.aws_organizations_organization.cool.accounts :
+    account.id
+    if length(regexall("^env\\d+", account.name)) > 0
```

Comment:
Are you planning ahead here for the near future when our assessment env account names change from (for example) `env123 (Staging)` to just `env123`?

Reply:
I updated the comment to mention this.  See commit e001ac8.

Reply:
Good call. 👍